src/NUnitFramework/framework/Internal/Commands/TimeoutCommand.cs

```diff
@@ -77,11 +80,40 @@ public TimeoutCommand(TestCommand innerCommand, int timeout)
                 // If the timer cancelled the current thread, change the result
                 if (_commandTimedOut)
                 {
-                    context.CurrentResult.SetResult(ResultState.Failure,
-                        string.Format("Test exceeded Timeout value of {0}ms", timeout));
+                    context.CurrentResult.SetResult(ResultState.Failure, $"Test exceeded Timeout value of {timeout}ms");
                 }
             };
+#else
+            BeforeTest = _ => { };
+            AfterTest = _ => { };
```

Comment:
ℹ The need for this is something I'd like to clean up in a future PR.

Reply:
Ok. Let us leave this for a future PR.